extensions/wikia/Recirculation/js/helpers/DataHelper.js

```diff
@@ -6,9 +6,19 @@ define('ext.wikia.recirculation.helpers.data', [
 	'wikia.nirvana'
 ], function ($, w, abTest, nirvana) {
 
-	return function() {
+	return function(options) {
+		options = options || {};
 
 		function loadData() {
+			switch(options.type) {
+				case 'articles':
+				default:
+					return loadArticles().then(function(articles) {
+						return { items: articles };
+					});
+			}
+
+
 			return $.when(loadAll(), loadArticles())
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this code after the "return" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnreachableCode)